src/main/webapp/encounters/encounter.jsp

```diff
@@ -834,7 +834,7 @@ $(function() {
             <div class="col-xs-12 col-sm-6" style="vertical-align: top;padding-left: 10px;">
 
 <%-- START LOCATION --%>
-<% if ((isOwner || isPublic) && CommonConfiguration.isCatalogEditable(context)) { %>
+<% if ((isOwner && CommonConfiguration.isCatalogEditable(context)) || (isPublic && !CommonConfiguration.isCatalogEditable(context))) { %>
```

Comment:
this should be returned to original code, except `(isOwner || isPublicEditable)`